### DISCUSSION TITLE
Lama github logo

### DISCUSSION THREAD

**Discussion by maplepy** [Sanster/IOPaint#310] [General]
Hey! I love the little lama logo and was wondering how you made it? Is this something with AI ? Or a blender render?

**Comment by maplepy** [comment]
Talking about this one: https://raw.githubusercontent.com/Sanster/lama-cleaner/main/assets/logo.png

**Comment by Sanster** [comment]
I did it using Midjourney.
![AFE03D1B-752E-4D6A-89DC-3D60E1B06220](https://github.com/Sanster/lama-cleaner/assets/3998421/2c431dce-dded-46c9-a885-3e007f02d435)

**Reply by maplepy** [reply]
Thanks ! It's so cute omg!!!
